examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -64,9 +65,27 @@
              n_jobs=1, proj=True, ax=ax, color=(0, 1, 0), picks=picks)
 
 # And now do the same with SSP + notch filtering
-raw.notch_filter(np.arange(60, 241, 60), picks=picks, n_jobs=1)
+# Pick all channels for notch since the SSP projection mixes channels together
+picks_notch = mne.pick_types(raw.info, meg=True, eeg=True)
```

Comment:
Yeah, I was thinking to add in picks_notch to be more explicit about what
I'm doing there. Think it's not necessary?

On Sat, Jan 2, 2016 at 11:04 AM, Eric Larson <notifications@github.com>
wrote:

> In examples/time_frequency/plot_compute_raw_data_spectrum.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r48685831>:
>
> > @@ -64,9 +65,27 @@
> >               n_jobs=1, proj=True, ax=ax, color=(0, 1, 0), picks=picks)
> >
> >  # And now do the same with SSP + notch filtering
> > -raw.notch_filter(np.arange(60, 241, 60), picks=picks, n_jobs=1)
> > +# Pick all channels for notch since the SSP projection mixes channels together
> > +picks_notch = mne.pick_types(raw.info, meg=True, eeg=True)
>
> FYI you don't actually need to provide picks_notch, because it's
> equivalent to picks=None in raw.filter (which means all data channels)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r48685831>.
>